api/proto/info/info.proto

```diff
@@ -504,9 +506,44 @@ message ZInfoNetworkInstance {
     ZInfoVpn vinfo = 31;
     ZInfoLisp linfo = 32;
   }
+  UsageInfo usage = 33;
   repeated ErrorInfo networkErr = 40;
 }
 
+message UsageInfo {
+  google.protobuf.Timestamp createTime = 1;
+  uint32 refCount = 2;
```

Comment:
Add comments? is this "How many apps are using the volumes?"

Reply:
The API doesn't dictate how a reference is held. It just captures that the number of references on the object. I'd rather not add specific implementation details in the comments, when API doesn't mandate it.

Reply:
UsageInfo is a generic object.. So refCount should be good enough.